<!doctype html><html lang=en><head><meta charset=utf-8><meta name=description content="Principles of Evolution January 28
Genetics Terms  Somatic cells - most cells in body, except sex cells Gametes - sex cells Cytoplasm - mix of membranes, molecules, and organelles Nucleus - contains hereditary material, or chromosomes Chromosomes - Structures in nucleus containing genes that transmit traits through generations  (humans have 23 pairs, other animals have more)    Critical Acids (DNA & RNA) Structure   Deoxyribonucleic Acid (DNA) - nucleic acid used to store genetic information"><title>ANT100: Lecture_16</title><meta name=viewport content="width=device-width,initial-scale=1"><link rel="shortcut icon" type=image/png href=https://mel10c.github.io/mel.zhu//icon.png><link href=https://mel10c.github.io/mel.zhu/styles.9a8661985b360a1d97e0e538e164abef.min.css rel=stylesheet><link href=https://mel10c.github.io/mel.zhu/styles/_light_syntax.86a48a52faebeaaf42158b72922b1c90.min.css rel=stylesheet id=theme-link><script src=https://mel10c.github.io/mel.zhu/js/darkmode.905e4d2da56a9111aff695a0a4b69900.min.js></script>
<script src=https://mel10c.github.io/mel.zhu/js/util.6f22941e242efae60fd84e7c32e874fa.min.js></script>
<link rel=preload href=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.css as=style onload='this.onload=null,this.rel="stylesheet"' integrity=sha384-R4558gYOUz8mP9YWpZJjofhk+zx0AS11p36HnD2ZKj/6JR5z27gSSULCNHIRReVs crossorigin=anonymous><script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/katex.min.js integrity=sha384-z1fJDqw8ZApjGO3/unPWUPsIymfsJmyrDVWC8Tv/a1HeOtGmkwNd/7xUS0Xcnvsx crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.15.1/dist/contrib/auto-render.min.js integrity=sha384-+XBljXPPiv+OzfbB3cVmLHf4hdUFHlWNZN5spNQ7rmHTXpd7WvJum6fIACpNNfIR crossorigin=anonymous></script>
<script defer src=https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/contrib/copy-tex.min.js integrity=sha384-ww/583aHhxWkz5DEVn6OKtNiIaLi2iBRNZXfJRiY1Ai7tnJ9UXpEsyvOITVpTl4A crossorigin=anonymous></script>
<script async src=https://unpkg.com/@floating-ui/core@0.7.3></script>
<script async src=https://unpkg.com/@floating-ui/dom@0.5.4></script>
<script async src=https://mel10c.github.io/mel.zhu/js/popover.f03552ccb84d99ca615d1cfb9abde59e.min.js></script>
<script defer src=https://mel10c.github.io/mel.zhu/js/code-title.ce4a43f09239a9efb48fee342e8ef2df.min.js></script>
<script defer src=https://mel10c.github.io/mel.zhu/js/clipboard.2913da76d3cb21c5deaa4bae7da38c9f.min.js></script>
<script defer src=https://mel10c.github.io/mel.zhu/js/callouts.7723cac461d613d118ee8bb8216b9838.min.js></script>
<script>const SEARCH_ENABLED=!1,LATEX_ENABLED=!0,PRODUCTION=!0,BASE_URL="https://mel10c.github.io/mel.zhu/",fetchData=Promise.all([fetch("https://mel10c.github.io/mel.zhu/indices/linkIndex.5702adc893417d7a266d9ed2e8ebdba1.min.json").then(e=>e.json()).then(e=>({index:e.index,links:e.links})),fetch("https://mel10c.github.io/mel.zhu/indices/contentIndex.29e8901604d17bba5377fc5ce2a7f2c6.min.json").then(e=>e.json())]).then(([{index:e,links:t},n])=>({index:e,links:t,content:n})),render=()=>{const e=new URL(BASE_URL),t=e.pathname,n=window.location.pathname,s=t==n;addCopyButtons(),addTitleToCodeBlocks(),addCollapsibleCallouts(),initPopover("https://mel10c.github.io/mel.zhu",!0);const o=document.getElementById("footer");if(o){const e=document.getElementById("graph-container");if(!e)return requestAnimationFrame(render);e.textContent="";const t=s&&!1;drawGraph("https://mel10c.github.io/mel.zhu",t,[{"/moc":"#4388cc"}],t?{centerForce:1,depth:-1,enableDrag:!0,enableLegend:!1,enableZoom:!0,fontSize:.5,linkDistance:1,opacityScale:3,repelForce:1,scale:1.4}:{centerForce:1,depth:1,enableDrag:!0,enableLegend:!1,enableZoom:!0,fontSize:.6,linkDistance:1,opacityScale:3,repelForce:2,scale:1.2})}var i=document.getElementsByClassName("mermaid");i.length>0&&import("https://unpkg.com/mermaid@9/dist/mermaid.esm.min.mjs").then(e=>{e.default.init()})},init=(e=document)=>{addCopyButtons(),addTitleToCodeBlocks(),renderMathInElement(e.body,{delimiters:[{left:"$$",right:"$$",display:!0},{left:"$",right:"$",display:!1}],macros:{'’':"'"},throwOnError:!1})}</script><script type=module>
    import { attachSPARouting } from "https:\/\/mel10c.github.io\/mel.zhu\/js\/router.9d4974281069e9ebb189f642ae1e3ca2.min.js"
    attachSPARouting(init, render)
  </script></head><body><div id=search-container><div id=search-space><input autocomplete=off id=search-bar name=search type=text aria-label=Search placeholder="Search for something..."><div id=results-container></div></div></div><script src=https://cdn.jsdelivr.net/npm/flexsearch@0.7.21/dist/flexsearch.bundle.js integrity="sha256-i3A0NZGkhsKjVMzFxv3ksk0DZh3aXqu0l49Bbh0MdjE=" crossorigin=anonymous defer></script>
<script defer src=https://mel10c.github.io/mel.zhu/js/full-text-search.e6e2e0c213187ca0c703d6e2c7a77fcd.min.js></script><div class=singlePage><header><h1 id=page-title><a href=https://mel10c.github.io/mel.zhu/>MEL.ZHU</a></h1><div class=spacer></div><div id=search-icon><p>Search</p><svg tabindex="0" aria-labelledby="title desc" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 19.9 19.7"><title id="title">Search Icon</title><desc id="desc">Icon to open search</desc><g class="search-path" fill="none"><path stroke-linecap="square" d="M18.5 18.3l-5.4-5.4"/><circle cx="8" cy="8" r="7"/></g></svg></div><div class=darkmode><input class=toggle id=darkmode-toggle type=checkbox tabindex=-1>
<label id=toggle-label-light for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="dayIcon" viewBox="0 0 35 35" style="enable-background:new 0 0 35 35"><title>Light Mode</title><path d="M6 17.5C6 16.672 5.328 16 4.5 16h-3C.672 16 0 16.672.0 17.5S.672 19 1.5 19h3C5.328 19 6 18.328 6 17.5zM7.5 26c-.414.0-.789.168-1.061.439l-2 2C4.168 28.711 4 29.086 4 29.5 4 30.328 4.671 31 5.5 31c.414.0.789-.168 1.06-.44l2-2C8.832 28.289 9 27.914 9 27.5 9 26.672 8.329 26 7.5 26zm10-20C18.329 6 19 5.328 19 4.5v-3C19 .672 18.329.0 17.5.0S16 .672 16 1.5v3C16 5.328 16.671 6 17.5 6zm10 3c.414.0.789-.168 1.06-.439l2-2C30.832 6.289 31 5.914 31 5.5 31 4.672 30.329 4 29.5 4c-.414.0-.789.168-1.061.44l-2 2C26.168 6.711 26 7.086 26 7.5 26 8.328 26.671 9 27.5 9zM6.439 8.561C6.711 8.832 7.086 9 7.5 9 8.328 9 9 8.328 9 7.5c0-.414-.168-.789-.439-1.061l-2-2C6.289 4.168 5.914 4 5.5 4 4.672 4 4 4.672 4 5.5c0 .414.168.789.439 1.06l2 2.001zM33.5 16h-3c-.828.0-1.5.672-1.5 1.5s.672 1.5 1.5 1.5h3c.828.0 1.5-.672 1.5-1.5S34.328 16 33.5 16zM28.561 26.439C28.289 26.168 27.914 26 27.5 26c-.828.0-1.5.672-1.5 1.5.0.414.168.789.439 1.06l2 2C28.711 30.832 29.086 31 29.5 31c.828.0 1.5-.672 1.5-1.5.0-.414-.168-.789-.439-1.061l-2-2zM17.5 29c-.829.0-1.5.672-1.5 1.5v3c0 .828.671 1.5 1.5 1.5s1.5-.672 1.5-1.5v-3C19 29.672 18.329 29 17.5 29zm0-22C11.71 7 7 11.71 7 17.5S11.71 28 17.5 28 28 23.29 28 17.5 23.29 7 17.5 7zm0 18c-4.136.0-7.5-3.364-7.5-7.5s3.364-7.5 7.5-7.5 7.5 3.364 7.5 7.5S21.636 25 17.5 25z"/></svg></label><label id=toggle-label-dark for=darkmode-toggle tabindex=-1><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" id="nightIcon" viewBox="0 0 100 100" style="enable-background='new 0 0 100 100'"><title>Dark Mode</title><path d="M96.76 66.458c-.853-.852-2.15-1.064-3.23-.534-6.063 2.991-12.858 4.571-19.655 4.571C62.022 70.495 50.88 65.88 42.5 57.5 29.043 44.043 25.658 23.536 34.076 6.47c.532-1.08.318-2.379-.534-3.23-.851-.852-2.15-1.064-3.23-.534-4.918 2.427-9.375 5.619-13.246 9.491-9.447 9.447-14.65 22.008-14.65 35.369.0 13.36 5.203 25.921 14.65 35.368s22.008 14.65 35.368 14.65c13.361.0 25.921-5.203 35.369-14.65 3.872-3.871 7.064-8.328 9.491-13.246C97.826 68.608 97.611 67.309 96.76 66.458z"/></svg></label></div></header><article><h1>ANT100: Lecture_16</h1><p class=meta>Last updated
Unknown</p><ul class=tags><li><a href=https://mel10c.github.io/mel.zhu/tags/note/>Note</a></li></ul><aside class=mainTOC><details><summary>Table of Contents</summary><nav id=TableOfContents><ol><li><a href=#genetics-terms>Genetics Terms</a><ol><li><a href=#critical-acids-dna--rna-structure>Critical Acids (DNA & RNA) Structure</a></li></ol></li><li><a href=#genetics-and-heredity>Genetics and Heredity</a><ol><li><a href=#adaptation-process-and-feature>Adaptation: process and feature</a></li><li><a href=#cladistics-a-biological-taxonomy-sytem>Cladistics: a biological ==taxonomy sytem==</a></li></ol></li></ol></nav></details></aside><a href=#principles-of-evolution><h1 id=principles-of-evolution><span class=hanchor arialabel=Anchor># </span>Principles of Evolution</h1></a><p><code>January 28</code></p><a href=#genetics-terms><h2 id=genetics-terms><span class=hanchor arialabel=Anchor># </span>Genetics Terms</h2></a><ul><li><u>Somatic cells</u> - most cells in body, except sex cells</li><li><u>Gametes</u> - sex cells</li><li><u>Cytoplasm</u> - mix of membranes, molecules, and organelles</li><li><u>Nucleus</u> - contains hereditary material, or chromosomes</li><li><strong><u>Chromosomes</u></strong> - Structures in nucleus containing genes that transmit traits through generations<ul><li><em>(humans have 23 pairs, other animals have more)</em></li></ul></li></ul><a href=#critical-acids-dna--rna-structure><h3 id=critical-acids-dna--rna-structure><span class=hanchor arialabel=Anchor># </span>Critical Acids (DNA & RNA) Structure</h3></a><ul><li><p><u>Deoxyribonucleic Acid (DNA)</u> - nucleic acid used to <strong>store genetic information</strong></p><ul><li>Four bases: Adenine (A), Guanine (G), Cytosine (C), and Thymine (T)</li><li>T bonds with A, G bonds with C</li></ul></li><li><p><u>Ribonucleic Acid Molecules (RNA)</u></p><ol><li>Dictate synthesis of proteins that perform a wide variety of functions in body</li><li><strong>Regulate expression</strong> of other genes</li><li>Work with structures in cell (ribosomes) that are critical for <strong>manufacture of proteins</strong></li><li><strong>Transport</strong> amino acids to ribosomes for the creation of proteins</li></ol></li><li><p><u>Proteins</u>: linear sequence of <strong>amino acids</strong> (<em>building blocks of cells</em>)</p><ul><li>Function determined by DNA; for example, <u><strong>enzymes</strong></u> (catalysis of biochemical reactions) or <u>hemoglobin</u> (transports oxygen in the blood)</li><li>Many animals (like dogs) cannot process caffeine or chocolate, as they lack the necessary <em>enzymes</em></li></ul></li><li><p><u>Transcription</u> - synthesis of single strand of <strong>mRNA (messenger RNA)</strong>; results in genetic information transferred from DNA to RNA, then carried to cytoplasm</p></li><li><p><u>Codons</u> - sequence of 3 nucleotides (basic structure unit for DNA/RNA) containing genetic info</p></li><li><p>4 <strong><u>nucleotides</u></strong> of RNA are: adenine (A), guanine (G), cytosine (C), and <strong><mark>uracil (U)</mark></strong> [which replaces thymine (T) in DNA template (uracil replaces thymine for increased RNA stability and replication efficiency)]</p></li><li><p><u>Translation</u> - <strong>tRNA (transfer RNA)</strong> (information adapter molecule) decodes information in mRNA; amino acid attaches to acceptor stem, then anticodon reads information via base pairing</p></li><li><p><code>DNA -> (Transcription) -> RNA -> (Translation) -> Protein</code></p></li></ul><a href=#genetics-and-heredity><h2 id=genetics-and-heredity><span class=hanchor arialabel=Anchor># </span>Genetics and Heredity</h2></a><ul><li><p><u>Gene</u> - chemical unit of heredity</p></li><li><p><u>Alleles</u> - one member of a pair of genes</p></li><li><p><mark><strong><u>Phenotype</u></strong></mark> - <strong>observable</strong> physical appearance of organism</p></li><li><p><u><strong>Genotype</strong></u> - total complement of inherited traits or genes</p></li><li><p><strong><mark><u>Homozygous</u></mark></strong> - possessing <strong>two identical genes</strong> or alleles in corresponding locations on a pair of chromosomes. For example, YY or yy</p></li><li><p>**<mark><u>Heterozygous</u></mark> ** - possessing <strong>different genes</strong> or alleles in corresponding locations on a pair of chromosomes. For example: Yy</p></li><li><p><strong><u>Dominant alleles</u></strong> - allele of gene pair that is <strong>always phenotypically expressed</strong> in heterozygous form [For example: Y always expressed phenotypically when paired with y (Yy)]</p></li><li><p><strong><u>Recessive alleles</u></strong> - allele <strong>phenotypically suppressed</strong> in heterozygous form and <em>expressed only in homozygous form</em>. [For example: y only expressed phenotypically when paired with y (yy)]</p><blockquote><p>In Mendel’s experiment (3 yellow: 1 green phenotypically) (1 YY: 2 Yy: 1 yy genotypically)</p></blockquote></li><li><p><u>Mutation</u> - randomly occurring; <strong>result of error/change in nucleotide sequence</strong>; somatic vs germ cell mutations</p><ul><li>Can be neutral, harmful, or (very rarely) beneficial; ultimate source of new genetic materials</li><li><em>4 factors:</em> ① <strong>copying errors</strong> in cell division, ② <strong>exposure to radiation</strong> (especially sunlight), ③ <strong>exposure to mutagens</strong>, or ④ <strong>exposure to viruses</strong></li></ul></li><li><p><u>Genetic Drift</u> - random changes in gene pool over time</p></li><li><p><em>3 important outcomes:</em> ① <strong>reduces within-population genetic variation</strong>, ② more likely to effect small populations, ③ <strong>increase between-population genetic variation</strong></p></li><li><p><u>Gene Flow</u> - movement of genes between populations</p></li><li><p><em>2 important outcomes:</em> ① initially, <strong>increases</strong> within-population genetic variation, ② eventually, <strong>reduces between-population genetic variation</strong></p></li><li><p><strong><u>Natural Selection</u></strong> - any consistent difference in fitness among <strong>phenotypically different</strong> biological entities; deterministic process; acts only on existing variation [<strong>Deterministic for evolution</strong>]</p><ul><li><mark><strong>Biological evolution can occur without natural selection, and vice versa</strong></mark></li></ul></li></ul><a href=#adaptation-process-and-feature><h3 id=adaptation-process-and-feature><span class=hanchor arialabel=Anchor># </span>Adaptation: process and feature</h3></a><ul><li><em>Process</em>: <strong>change</strong> in organism enabling it to better reproduce and survive in its environment</li><li><em>Feature</em>: <strong>characteristic</strong> that performs a function of utility to the organism possessing it</li><li><u>Antimicrobial Resistance (AMR)</u> - serious threat to global public health; 480 000 people develop drug-resistant TB each year</li><li><u>Macroevolution</u> - large-scale changes at or above the species level; extends over geological era</li></ul><a href=#cladistics-a-biological-taxonomy-sytem><h3 id=cladistics-a-biological-taxonomy-sytem><span class=hanchor arialabel=Anchor># </span>Cladistics: a biological <mark>taxonomy sytem</mark></h3></a><ul><li><p>Based on <em>qualitative analysis</em> of comparative data that is used to reconstruct (assumed) phylogenetic relationships and evolutionary history of groups of organisms</p></li><li><p><u>Clade</u> - a group of organisms that contains an <strong>ancestral <u>taxon</u></strong> (group of one/more organism) and all of its <strong>descendants</strong>; can orient cladogram in any direction with no change</p></li><li><p><u>Tree topology</u> - the branching patterns of lines connecting nodes and organisms</p><ul><li>Focus on internal branches; watch out for polytomies</li></ul></li><li><p><u>Cladogram</u> - branching <em>diagram</em> used to illustrate phylogenetic relationships</p><ul><li>May have any number of clades</li><li>Reading a Cladogram - Interested in tree topology;</li></ul></li><li><a href=#3-major-assumptions-of-cladistics><h5 id=3-major-assumptions-of-cladistics><span class=hanchor arialabel=Anchor># </span>3 Major Assumptions of cladistics</h5></a><ul><li>There are changes in characteristics within lineages <strong>over time</strong> (Some have not changed in many years, like Greenland sharks or coelacanths)</li><li>All organisms are descended from a <strong>common ancestor</strong></li><li>When a lineage splits, it <strong>divides into exactly two groups</strong></li></ul></li><li><a href=#how-to-do-cladistics><h5 id=how-to-do-cladistics><span class=hanchor arialabel=Anchor># </span>How to do Cladistics:</h5></a><ol><li>Choose taxa whose evolutionary relationships interest you. These taxa must be <u>clades</u> (monophyletic group) if you hope to come up with plausible results</li><li>Determine <u>characters</u> (features of organisms) and examine each taxon to determine character <u>states</u> (decide whether each taxon does or does not have each character)</li><li>Determine polarity of characters (character state is original or derived in each taxon) using <u>out-group</u> (more distantly related group of organisms that serves as a reference group)</li><li>Group taxa by <u>synapomorphies</u> (shared derived characteristics; <strong>derived</strong>) not <u>symplesiomorphies</u> (original, or “primitive”, characteristics; <strong>ancestral</strong>)</li></ol></li><li><a href=#3-misconceptions-about-cladograms><h5 id=3-misconceptions-about-cladograms><span class=hanchor arialabel=Anchor># </span>3 Misconceptions About Cladograms</h5></a><ol><li>Evolution produces patterns of relationships among organisms that are like a tree</li><li>Although cladograms are often organized from top to bottom, <strong>don’t assume the taxa on top are more advanced than those taxa on the bottom</strong></li><li>Avoid reading across terminal nodes (tips), order of which has no meaning</li></ol></li></ul></article><hr><div class=page-end id=footer><div class=backlinks-container><h3>Backlinks</h3><ul class=backlinks><li>No backlinks found</li></ul></div><div><script async src=https://cdn.jsdelivr.net/npm/d3@6.7.0/dist/d3.min.js integrity="sha256-+7jaYCp29O1JusNWHaYtgUn6EhuP0VaFuswhNV06MyI=" crossorigin=anonymous></script><h3>Interactive Graph</h3><div id=graph-container></div><style>:root{--g-node:var(--secondary);--g-node-active:var(--primary);--g-node-inactive:var(--visited);--g-link:var(--outlinegray);--g-link-active:#5a7282}</style><script src=https://mel10c.github.io/mel.zhu/js/graph.2d9e48dbe7ea47c0ef1c58296ce14448.js></script></div></div><div id=contact_buttons><footer><p>Made by Melaney Zhu using <a href=https://github.com/jackyzha0/quartz>Quartz</a>, © 2023</p><ul><li><a href=https://mel10c.github.io/mel.zhu/>Home</a></li><li><a href=https://www.instagram.com/melaney_dxl/>Instagram</a></li><li><a href=https://github.com/mel10c>GitHub</a></li><li><a href=https://linkedin.com/in/melzyy>LinkedIn</a></li><li><a href=https://github.com/mel10c/mel.zhu/blob/hugo/content/documents/Resume.pdf>Resume</a></li></ul></footer></div></div></body></html>